airflow/providers/amazon/aws/triggers/rds.py

```diff
@@ -99,13 +99,14 @@ async def run(self):
         yield TriggerEvent({"status": "success", "response": self.response})
 
 
-_waiter_arg = {
-    RdsDbType.INSTANCE: "DBInstanceIdentifier",
-    RdsDbType.CLUSTER: "DBClusterIdentifier",
+# using Any for the keys  to "disable" type checks because Union types.
```

Comment:
Why does this need to disable type checking? Aren’t those keys just strings?

https://github.com/apache/airflow/blob/f70c10748b23d6f06eeb520b8854a87345cb9765/airflow/providers/amazon/aws/utils/rds.py#L22-L26

Reply:
That’s what I meant, since you use `.value`, isn’t this dict just `dict[str, str]`? Mypy should be smart enough to infer this automatically without the `Any`.

Reply:
it's because of the function. But it's mostly coming from the fact that I reuse a variable changing its type. I'll fix that, it'll be easier for everyone.